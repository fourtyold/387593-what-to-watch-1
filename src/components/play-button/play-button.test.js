import React from "react";
import renderer from "react-test-renderer";
import {HashRouter} from "react-router-dom";

import PlayButton from "./play-button.jsx";

it(`Play button renders correctly`, () => {
  const tree = renderer.create(
      <HashRouter>
        <PlayButton/>
      </HashRouter>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
