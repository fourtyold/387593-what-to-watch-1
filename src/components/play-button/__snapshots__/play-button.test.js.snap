// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Play button renders correctly 1`] = `
<a
  className="btn btn--play movie-card__button"
  href="#/film/undefined/player"
  onClick={[Function]}
>
  <svg
    height="19"
    viewBox="0 0 19 19"
    width="19"
  >
    <use
      xlinkHref="#play-s"
    />
  </svg>
  <span>
    Play
  </span>
</a>
`;
